### PR TITLE
Instruct to deploy kube-state-metrics manually

### PR DESCRIPTION
**Pull request by yyyogev**
Signed-off-by: yyyogev <yogev.metzuyanim@logz.io>

# What changed

Instruct to deploy kube-state-metrics manually both in  the script and the manual instructions

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Comment by yyyogev** [comment]
A fix for a [ticket](https://logzio.atlassian.net/browse/IN-19)

**Comment by schwin007** [comment]
@yyyogev did you remove the deployment of kube-state-metrics from the automatic deployment script as well?

**Comment by yyyogev** [comment]
Yes

File: shipping-config-samples/quickstart.sh [hunk 1]
```diff
@@ -20,9 +20,8 @@ fi
 
 has_kube_stat_metrics=$(kubectl get deployments --all-namespaces | grep kube-state-metrics)
 if [[ -z $has_kube_stat_metrics ]]; then
-  git clone https://github.com/kubernetes/kube-state-metrics.git
-  kubectl --namespace=kube-system apply -f kube-state-metrics/examples/standard
-  rm  -rf kube-state-metrics
+  echo "ERROR: kube-state-metrics is not deployed in your cluster, please deploy it from https://github.com/kubernetes/kube-state-metrics and run this script again"
+  exit 1
```
Comment (shalper) [inline shipping-config-samples/quickstart.sh:24]: @imnotashrimp  - I'm not familiar with this file. Please be in touch.

File: shipping-config-samples/quickstart.sh [hunk 2]
```diff
@@ -20,9 +20,8 @@ fi
 
 has_kube_stat_metrics=$(kubectl get deployments --all-namespaces | grep kube-state-metrics)
 if [[ -z $has_kube_stat_metrics ]]; then
-  git clone https://github.com/kubernetes/kube-state-metrics.git
-  kubectl --namespace=kube-system apply -f kube-state-metrics/examples/standard
-  rm  -rf kube-state-metrics
+  echo "ERROR: kube-state-metrics is not deployed in your cluster, please deploy it from https://github.com/kubernetes/kube-state-metrics and run this script again"
```
Comment (imnotashrimp) [inline shipping-config-samples/quickstart.sh:23]:
```suggestion
  echo "ERROR: kube-state-metrics is not deployed in your cluster. Please deploy it from https://github.com/kubernetes/kube-state-metrics and run this script again"
```

**Comment by imnotashrimp** [comment]
Sweet beans @yyyogev

**Pull request merged**